CRM/Pledge/BAO/PledgeBlock.php

```diff
@@ -281,11 +281,11 @@ public static function buildPledgeBlock($form) {
       $form->addRadio('is_pledge', ts('Pledge Frequency Interval'), $pledgeOptions,
         NULL, array('<br/>')
       );
-      $form->addElement('text', 'pledge_installments', ts('Installments'), array('size' => 3));
+      $form->addElement('text', 'pledge_installments', ts('Installments'), array('size' => 3, 'aria-label' => ts('Pledge installements')));
```

Comment:
Rather than add an extra title why don't we make the addElement function add the title as an aria-label?